lms/djangoapps/teams/views.py

```diff
@@ -1183,12 +1182,12 @@ class MembershipListView(ExpandableFieldViewMixin, GenericAPIView):
 
             * username: Returns membership records only for the specified user.
               If the requesting user is not staff then only memberships for
-              teams associated with courses in which the requesting user is
+              teams associated with teamsets in which the requesting user is
               enrolled are returned.
 
             * team_id: Returns only membership records associated with the
-              specified team. The requesting user must be staff or enrolled in
-              the course associated with the team.
+              specified team. The requesting user must be staff or a member of
```

Comment:
This is incorrect. A user could request info about a team in an open teamset without being enrolled in the teamset. The visibility requiremnts are a bit more complex than this comment would imply, but the change is wrong.

Reply:
Reverted.